packages/apps/human-app/frontend/src/pages/operator/sign-up/add-keys/edit-existing-keys-form.tsx

```diff
@@ -64,7 +64,10 @@ const formInputsConfig: Record<EthKVStoreKeyValues, React.ReactElement> = {
     <MultiSelect
       label={t('operator.addKeysPage.existingKeys.jobType')}
       name={EthKVStoreKeys.JobTypes}
-      options={JOB_TYPES}
+      options={JOB_TYPES.map((jobType) => ({
```

Comment:
Decided to keep constants for operator to make it intuitive for them what to pick